File: scripts/fix-lockfile.js
import { execSync } from 'child_process';

try {
  console.log('Regenerating pnpm-lock.yaml...');
  execSync('pnpm install --no-frozen-lockfile', {
    cwd: '/vercel/share/v0-project',
    stdio: 'inherit'
  });
  console.log('✓ pnpm-lock.yaml has been regenerated successfully!');
} catch (error) {
  console.error('Error regenerating lockfile:', error.message);
  process.exit(1);
}

File: scripts/fix-lockfile.py
import subprocess
import os

os.chdir('/vercel/share/v0-project')

print('[v0] Regenerating pnpm-lock.yaml...')

try:
    result = subprocess.run(['pnpm', 'install'], capture_output=True, text=True)
    print(result.stdout)
    if result.returncode != 0:
        print('[v0] Error:', result.stderr)
    else:
        print('[v0] Successfully regenerated pnpm-lock.yaml')
except Exception as e:
    print(f'[v0] Error: {str(e)}')
